cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```diff
@@ -16,37 +16,52 @@ parts:
       - enable-patchelf
     stage-packages:
       - clang-12
-      - cmake
-      - libc-dev
-      - libclang-cpp12-dev
+      - libatomic1
+      - libcurl4-openssl-dev
       - libdbus-1-3
-      - libfontconfig1-dev
-      - libglu1-mesa-dev
+      - libffi7
```

Comment:
`libffi7` is still needed?

Reply:
It came from the deb package dependencies, but now I recall that we don't need it anymore from: https://github.com/o3de/o3de/issues/15923. I'll remove